Gems/ROS2/gem.json

```diff
@@ -0,0 +1,21 @@
+{
+    "gem_name": "ROS2",
+    "display_name": "ROS2",
+    "license": "Apache-2.0",
+    "license_url": "https://opensource.org/licenses/Apache-2.0",
+    "origin": "RobotecAI",
+    "origin_url": "https://github.com/RobotecAI/o3de-ros2-gem",
```

Comment:
This will need to be updated to https://github.com/o3de/o3de-extras/development/Gems/ROS2